sdk/src/main/java/com/mapbox/maps/renderer/MapboxWidgetRenderer.kt

```diff
@@ -18,7 +19,7 @@ internal class MapboxWidgetRenderer(
   private val textures = intArrayOf(0)
   private val framebuffers = intArrayOf(0)
 
-  private val widgets = mutableListOf<Widget>()
+  private val widgets = CopyOnWriteArrayList<Widget>()
```

Comment:
```suggestion
  private val widgets = CopyOnWriteArraySet<Widget>()
```

Are there tests for this by chance? Seems like copy on array set would be better, because when would you want multiple identical widgets?

With a list the `fun removeWidget(widget: Widget) = widgets.remove(widget)` is ambiguous

Reply:
sounds reasonable to me, will apply, thanks!